enderio-decoration/src/main/java/com/enderio/decoration/datagen/model/block/DecorBlockState.java

```diff
@@ -19,4 +26,21 @@ public static void paintedBlock(DataGenContext<Block, ? extends Block> ctx, Regi
         prov.models().generatedModels.put(paintedModel.getLocation(), paintedModel);
         prov.simpleBlock(paintedBlock, paintedModel);
     }
+    
+    public static void lightBlock(DataGenContext<Block, ? extends Block> ctx, RegistrateBlockstateProvider prov) {
+    	prov.getVariantBuilder(ctx.get()).forAllStates(state -> {
```

Comment:
Look at your formatting in this method :)

Reply:
I'm not to sure what you mean?